gui/system/views.py

```diff
@@ -973,6 +973,58 @@ def file_browser(request, path='/'):
     return HttpResponse(content, content_type='application/json')
 
 
+def manualupdate_running(request):
+    uuid = request.GET.get('uuid')
+    if not uuid:
+        return HttpResponse(uuid, status=202)
+
+    _n = notifier()
+    if not _n.is_freenas() and _n.failover_licensed():
+        if uuid != "0":
```

Comment:
`uuid` is an int above, here you're matching a string?

Reply:
uuid is a str here, not GET params are int, not possible.